simbad2k/simbad2k.py

```diff
@@ -96,67 +103,120 @@ def __init__(self, query, scheme):
         self.scheme = scheme
 
     def _clean_result(self, result):
+        """
+        Converts the results from the MPC into a dictionary of floats.
+        Extracts the object's name from the query results and adds it to the dictionary.
+        """
         cleaned_result = {}
         for key in self.keys:
             try:
                 value = float(result[key])
             except (ValueError, TypeError):
                 value = None
             cleaned_result[key] = value
+        # Build object name from returned Data
+        if result['number']:
+            if result.get('name',None):
```

Comment:
result.get('name') will return None by default if the key doesn't exist, so we probably don't need the extra None there.